### PR TITLE
Make privacy policy available without authentication

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/cht-core#7662

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: api/src/templates/privacy-policy/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  </head>
+  <body>
+    <div>
+      <h1>{{ translations.title }}</h1>
+    </div>
+    <div>
+      {{ policy }}
+    </div>
+    <div>
+      <% if(showBackButton) { %>
```
Comment (dianabarsan) [inline api/src/templates/privacy-policy/index.html:35]: Is this best practice, to put the back or login button at the bottom only? Thinking about very long privacy policy texts on mobile.

**Review by dianabarsan — APPROVED** [review]
Awesome work! I added a couple of very minor comments inline.

File: api/src/services/privacy-policy.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+const sanitizeHtml = require('sanitize-html');
+const db = require('../db');
+const logger = require('../logger');
+const config = require('../config');
+
+const PRIVACY_POLICY_DOC_ID = 'privacy-policies';
+
+const getAttachmentName = (doc, locale) => {
+  const policies = doc.privacy_policies;
+  return policies[locale] || Object.values(policies)[0];
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply api/src/services/privacy-policy.js:10]: I've added `policies['en']` with the first entry as the final backstop.

File: api/src/templates/privacy-policy/index.html (hunk 1 above)
Comment (garethbowen) [reply api/src/templates/privacy-policy/index.html:35]: Copied the section to the top of the doc too.

File: api/src/services/privacy-policy.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        if (err.status !== 404) {
+          logger.error('Error retrieving privacy policies: %o', err);
+        }
+        throw err;
+      });
+  },
+  exists: () => {
+    return getDoc()
+      .then(() => true)
+      .catch(() => false);
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply api/src/services/privacy-policy.js:46]: Added logging if the error is not a 404.

File: api/tests/mocha/services/privacy-policy.spec.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      });
+    });
+
+    it('defaults to configured default locale', () => {
+      sinon.stub(config, 'get').returns('ne');
+      sinon.stub(db.medic, 'get').resolves(doc);
+      return service.get().then(response => {
+        chai.expect(response).to.equal('test');
+      });
+    });
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply api/tests/mocha/services/privacy-policy.spec.js:94]: Done.

**Review by dianabarsan — APPROVED** [review]
Awesome!

**Pull request merged**